apps/webservice/src/app/[workspaceSlug]/_components/environment-policy-drawer/EnvironmentPolicyDrawer.tsx

```diff
@@ -96,14 +107,20 @@ const PolicyDropdownMenu: React.FC<{
 export const EnvironmentPolicyDrawer: React.FC = () => {
   const { environmentPolicyId, removeEnvironmentPolicyId } =
     useEnvironmentPolicyDrawer();
-  const isOpen = environmentPolicyId != null && environmentPolicyId != "";
+  const isOpen = Boolean(environmentPolicyId);
```

Comment:
I prefer explicit checks

Reply:
Boolean does more falsy checks on the string than what we had before and its more concise, so i'd argue its more explicit because it covers all the bases